envs/monkey_zoo/terraform/firewalls.tf

```
@@ -156,6 +156,21 @@ resource "google_compute_firewall" "powershell-48-allow" {
   target_tags = ["powershell-48"]
 }
 
+resource "google_compute_firewall" "block-powershel-48-to-depth-3 " {
+  name    = "${local.resource_prefix}block-powershel-48-to-depth-3 "
```

Ignore if this is more trouble than it's worth.
```suggestion
resource "google_compute_firewall" "block-powershell-48-to-depth-3 " {
  name    = "${local.resource_prefix}block-powershell-48-to-depth-3 "
```

---

Nice catch. It is not trouble at all!